pkgs/by-name/ju/jumpy/package.nix

```diff
@@ -26,13 +26,9 @@ rustPlatform.buildRustPackage rec {
     sha256 = "sha256-ggePJH2kKJ17aOWRKUnLyolIdSzlc6Axf5Iw74iFfek=";
   };
 
-  cargoLock = {
-    lockFile = ./Cargo.lock;
-    outputHashes = {
-      "bevy_egui-0.21.0" = "sha256-hu55tZQppw1NajwqIsYsw6de0IAwQwgra3D9OFzSSLc=";
-      "bones_asset-0.3.0" = "sha256-1UeOXW6O/gMQBBUnHxRreJgmiUTPC5SJB+uLn9V8aa4=";
-      "kira-0.8.5" = "sha256-z4R5aIaoRQQprL6JsVrFI69rwTOsW5OH01+jORS+hBQ=";
-    };
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit pname version src;
+    hash = "sha256-hVEpTNTXwOQoxlhOewUvHyfBh+APnx8Fox90CmdMRQ4=";
   };
```

Comment:
Why not this?

```suggestion
  useFetchCargoVendor = true;
  cargoHash = "sha256-hVEpTNTXwOQoxlhOewUvHyfBh+APnx8Fox90CmdMRQ4=";
```

Reply:
thanks. did not know this existed.

Reply:
It is pretty recent, it will probably be the default for `buildRustPackage` soon
https://github.com/NixOS/nixpkgs/blob/cf659838604b4809dbfbfa3519013eaefe3609fa/doc/release-notes/rl-2505.section.md?plain=1#L18-L20